### ISSUE TITLE
NTR: Accumulation of Dol-PP-GlcNAc2Man5 

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#4657]
**Preferred term label:**
Accumulation of Dol-PP-GlcNAc2Man5 in fibroblasts

**Synonyms**
Fibroblasts accumulate Dol-PP-GlcNAc2Man5

**Definition (free text, please give PubMed ID)**
A deficiency in the first dol-P-Man dependent mannosyltransferase, which leads to an accumulation of dol-PP-GlcNAc2Man5 in fibroblasts.
Source: PMID:16053906

**Parent term (use hpo.jax.org/app)**
HP_0011017 'Abnormal cellular physiology'
_(is this the best parent?)_

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://www.omim.org/entry/615042?search=615042&highlight=615042
(Note - I am reviewing the annotations for diseases listed in the GPI paper)

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0001-5208-3432

@pnrobinson I can add this if it looks okay

**Comment by pnrobinson** [comment]
See this for background
https://www.ncbi.nlm.nih.gov/pubmed/28575298

**Comment by pnrobinson** [comment]
Comment:

During protein N-glycosylation, glycans from lipid-linked oligosaccharides (LLOs) are transferred onto asparagine residues located in N-glycosylation sequons. The synthesis of LLO comprises three steps: It begins with the biosynthesis of the intermediate Dol-PP-GlcNAc2Man5, which occurs on the cytoplasmic side of the endoplasmic reticulum (ER). The second step is the flipping of this intermediate to the luminal side of the ER. The third step comprises the glycan extension in the ER lumen, yielding Dol-PP-GlcNAc2Man9 (Man9).

**Comment by pnrobinson** [comment]
Adding as HP_0032576
